pkg/cluster/constants/kafka/kcManager.go

```diff
@@ -0,0 +1,52 @@
+package kafka
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	AKCGroup   = "rhoas.redhat.com"
+	AKCVersion = "v1alpha1"
+)
+
+var AKCRMeta = metav1.TypeMeta{
+	Kind:       "KafkaConnection",
+	APIVersion: AKCGroup + "/" + AKCVersion,
+}
+
+var AKCResource = schema.GroupVersionResource{
+	Group:    AKCGroup,
+	Version:  AKCVersion,
+	Resource: "kafkaconnections",
+}
+
+var tokenSecretName = "rh-cloud-services-accesstoken-cli"
+
+/*  #nosec */
+var serviceAccountSecretName = "rh-cloud-services-service-account"
+
+func GetKafkaConnectionsAPIURL(namespace string) string {
+	return fmt.Sprintf("/apis/rhoas.redhat.com/v1alpha1/namespaces/%v/kafkaconnections", namespace)
+}
+
+func CreateKCObject(crName string, namespace string, kafkaID string) *KafkaConnection {
```

Comment:
I question value of this being in constants. 
It can be part of the service - is that used for some circular dependency avoidance?